app.py

```diff
@@ -23,6 +23,8 @@
     "networking",
     stack_name=f"geospatial-data-lake-networking-{ENV}",
     env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
+    deploy_env=ENV,
+    use_existing_vpc=True,  # use already existing VPC, pre-configured in AWS account
```

Comment:
As the code stands it's not possible to call this with `False` without changing the code. I'd probably do something like `use_existing_vpc = str2bool(os.environ.get("USE_EXISTING_VPC"))` where

```
def str2bool(value: str) -> bool:
    if value == "True":
        return True
    if value == "False":
        return False
    raise ValueError(f"Not a valid boolean: '{value}'")
```

Up to you if you want to do that in a separate PR.

Reply:
Yes , this is something which I am aware of. I was thinking about env variable as well.

Reply:
Should be done in d1e2abaeab5a4c9946386678b3b06a97a246ffbe